.infrastructure/helm-chart/todoapp/templates/app/pvc.yml

```diff
@@ -0,0 +1,13 @@
+
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Chart.Name }}-pvc-data
+  namespace: {{ .Values.common.namespace }}
+spec:
+  volumeName: {{ .Chart.Name }}-pv-data
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{ .Values.pvc.capacity }}
```

Comment:
Ensure that `.Values.pvc.capacity` is defined in your `values.yaml` file to specify the storage request for the PVC.